templates/systemd.service.erb

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Conntrack Daemon
+Documentation=man:conntrackd(8) man:conntrackd.conf(5)
+<%- if @systemd_before.length > 0 -%>
+Before=<%= @systemd_before.join(' ') %>
+<%- end -%>
+<%- if @systemd_after.length > 0 -%>
+After=<%= @systemd_after.join(' ') %>
+<%- end -%>
+<%- if @systemd_wants.length > 0 -%>
+Wants=<%= @systemd_wants.join(' ') %>
+<%- end -%>
+
+[Service]
+Type=notify
+ExecStart=/usr/sbin/conntrackd -C /etc/conntrackd/conntrackd.conf
```

Comment:
Should we add this parameter to ensure the existence of the config file ? 
`ConditionPathExists=`

https://www.freedesktop.org/software/systemd/man/systemd.unit.html#ConditionArchitecture=

Reply:
I would leave the unit without it, as if the config file does not exists, it will fail as the conntrackd process will fail, so there is no need to that extra check on systemd side.

Reply:
:+1: